### ISSUE TITLE
'NoneType' object is not iterable on Review step

### ISSUE THREAD

**Issue by derekeder** [datamade/dedupe-api#178]
```bash
Traceback (most recent call last):
  File "/Users/derekeder/.virtualenvs/dedupeapi3/lib/python3.4/site-packages/flask/app.py", line 1836, in __call__
    return self.wsgi_app(environ, start_response)
  File "/Users/derekeder/.virtualenvs/dedupeapi3/lib/python3.4/site-packages/flask/app.py", line 1820, in wsgi_app
    response = self.make_response(self.handle_exception(e))
  File "/Users/derekeder/.virtualenvs/dedupeapi3/lib/python3.4/site-packages/flask/app.py", line 1403, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/derekeder/.virtualenvs/dedupeapi3/lib/python3.4/site-packages/flask/_compat.py", line 33, in reraise
    raise value
  File "/Users/derekeder/.virtualenvs/dedupeapi3/lib/python3.4/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/derekeder/.virtualenvs/dedupeapi3/lib/python3.4/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/derekeder/.virtualenvs/dedupeapi3/lib/python3.4/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/derekeder/.virtualenvs/dedupeapi3/lib/python3.4/site-packages/flask/_compat.py", line 33, in reraise
    raise value
  File "/Users/derekeder/.virtualenvs/dedupeapi3/lib/python3.4/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/derekeder/.virtualenvs/dedupeapi3/lib/python3.4/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/derekeder/projects/datamade/dedupe-api/api/auth.py", line 123, in decorated
    return f(*args, **kwargs)
  File "/Users/derekeder/projects/datamade/dedupe-api/api/review.py", line 91, in get_canon_cluster
    'cr_{0}')
  File "/Users/derekeder/projects/datamade/dedupe-api/api/utils/helpers.py", line 265, in getCluster
    entity_id = bytes(machine.get_next()).decode('utf-8')
TypeError: 'NoneType' object is not iterable
```

**Comment by evz** [comment]
This is something I've encountered more recently. I am working on figuring this out over in the [``custom_dialect``](https://github.com/datamade/dedupe-api/tree/custom_dialect) branch

**Comment by evz** [comment]
Fixed with 041f40c8a50a831063796e727671678d3e0865e6
